src/main/java/io/confluent/connect/jdbc/sink/dialect/SqliteDialect.java

```diff
@@ -36,6 +40,15 @@ public SqliteDialect() {
 
   @Override
   protected String getSqlType(String schemaName, Map<String, String> parameters, Schema.Type type) {
+    if (schemaName != null) {
+      switch (schemaName) {
+        case Decimal.LOGICAL_NAME:
+        case Date.LOGICAL_NAME:
+        case Time.LOGICAL_NAME:
+        case Timestamp.LOGICAL_NAME:
+          return "NUMERIC";
```

Comment:
Does SQLite not need a custom implementation of `formatColumnValue`? My (very quick) reading of [the docs](https://www.sqlite.org/datatype3.html#date_and_time_datatype) suggests you might need to use the datetime methods to get things in the desired format? In particular, because sqlite seems to support flexible column formats, I'm unclear whether passing in strings and marking the column type `NUMERIC` will actually get you data in the format you expect. In particular, look at [the rules](https://www.sqlite.org/datatype3.html#type_affinity) for what happens if you pass in `TEXT` for a `NUMERIC` column....